0.10/s2i/bin/assemble

```diff
@@ -48,7 +48,10 @@ if [ -n "$NPM_MIRROR" ]; then
 fi
 
 echo "---> Building your Node application from source"
-npm install -d
+case $NPM_LOG_LEVEL in
+    silent|error|warn|http|info|verbose|silly) npm install --loglevel {$NMP_LOG_LEVEL};;
+    *)             npm install;;
```

Comment:
> so we're changing the default logging level from "info" to "warn"?

Yes.

AFAIK the initial motivation for this is that `npm install -d` was printing way more output than desired (and than the default).